releases/CHANGELOG-3.0.md

```diff
@@ -0,0 +1,205 @@
+  }, []);
+
+  return (
+    <div style={{ height: "100vh" }}>
+      <Viewer
+        authClient={authClient}
+        iTwinId={iTwinId ?? ""}
+        iModelId={iModelId ?? ""}
+        enablePerformanceMonitors={true}
+        onIModelAppInit={onIModelAppInit}
+        uiProviders={[
+          new ViewerNavigationToolsProvider(),
+          new ViewerContentToolsProvider({
+            vertical: {
+              measureGroup: false,
+            },
+          }),
+          new ViewerStatusbarItemsProvider(),
+          new TreeWidgetUiItemsProvider(),
+          new PropertyGridUiItemsProvider({
+            enableCopyingPropertyText: true,
+          }),
+          new MeasureToolsUiItemsProvider(),
+        ]}
+      />
+    </div>
+  );
+};
+```
+
+- Removed support for Azure Application Insights
+
+### New minimum iTwin.js version
+
+- With the addition of support for [iTwin.js extensions](#itwinjs-extensions), which were introduced in iTwin.js version 3.2.0, the Viewer packages now require that version as the minimum version of all iTwin.js peerDependencies (packages in the `@itwin` scope)
+
+## New Features
+
+### iTwin.js Extensions
+
+- While still in alpha and subject to change, iTwin Viewer 3.0 adds support for local and remote iTwin.js Extensions:
+
+```tsx
+import { BrowserAuthorizationClient } from "@itwin/browser-authorization";
+import LocalExtension from "@itwin/test-local-extension";
+import { Viewer } from "@itwin/web-viewer-react";
+import React, { useCallback, useMemo, useState } from "react";
+
+export const App: React.FC = () => {
+  const [iModelId, setIModelId] = useState(
+    process.env.IMJS_AUTH_CLIENT_IMODEL_ID
+  );
+  const [iTwinId, setITwinId] = useState(process.env.IMJS_AUTH_CLIENT_ITWIN_ID);
+
+  const authClient = useMemo(
+    () =>
+      new BrowserAuthorizationClient({
+        scope: process.env.IMJS_AUTH_CLIENT_SCOPES ?? "",
+        clientId: process.env.IMJS_AUTH_CLIENT_CLIENT_ID ?? "",
+        redirectUri: process.env.IMJS_AUTH_CLIENT_REDIRECT_URI ?? "",
+        postSignoutRedirectUri: process.env.IMJS_AUTH_CLIENT_LOGOUT_URI,
+        responseType: "code",
+        authority: process.env.IMJS_AUTH_AUTHORITY,
+      }),
+    []
+  );
+
+  return (
+    <div style={{ height: "100vh" }}>
+      <Viewer
+        authClient={authClient}
+        iTwinId={iTwinId ?? ""}
+        iModelId={iModelId ?? ""}
+        enablePerformanceMonitors={true}
+        extensions={[
+          new LocalExtensionProvider({
+            manifestPromise: LocalExtension.manifestPromise,
+            main: LocalExtension.main,
+          }),
+          new RemoteExtensionProvider({
+            jsUrl: "http://localhost:3001/dist/index.js",
+            manifestUrl: "http://localhost:3001/package.json",
+          }),
+        ]}
+      />
+    </div>
+  );
+};
+```
+
+### Loading Bar
+
+- We've changed the style of the default loading bar that displays when an iModel is loading
+
+### renderSys configuration
```

Comment:
i believe we added this is 2.x

Reply:
i think we just exposed the realityDataAccess,

renamed the tileAdmin prop to match whats in core
most of the changes i think can be seen in types.ts

Reply:
do you think we need to explicitly call out renaming `tileAdminProps` -> `tileAdmin` and changing the type for `defaultUiConfig` ? Should be pretty apparent when you look at the .d.ts file so not sure if worth mentioning, but thoughts?